soft4pes/control/mpc/controllers/im_mpc_curr_ctr.py

```diff
@@ -21,7 +34,7 @@ class IMMpcCurrCtr:
         Prediction horizon steps.
     Ts : float
         Sampling time [s].
-    T_ref_seq : Sequence
+    T_ref_seq : Sequence object
```

Comment:
``Ts`` is the sampling interval (not sampling time). Let's try to be consistent throughout

Reply:
Fixed in the next commit